src/main/java/jimmy/mcgymmy/logic/commands/ImportCommand.java

```diff
@@ -11,14 +11,17 @@
 import jimmy.mcgymmy.model.ReadOnlyMcGymmy;
 import jimmy.mcgymmy.storage.JsonMcGymmyStorage;
 
+/**
+ * Changes the data in mcgymmy to the data in the specified file.
+ * File must be available and in correct format
+ */
 public class ImportCommand extends Command {
 
     public static final String COMMAND_WORD = "import";
     public static final String SHORT_DESCRIPTION = "Import a McGymmy Save file";
 
     public static final String MESSAGE_IMPORT_FOOD_SUCCESS = "Imported %s";
     public static final String MESSAGE_IMPORT_FOOD_FAILURE = "File is invalid";
-    public static final String FILE_CONSTRAINTS = "Please select a valid .json file";
```

Comment:
This is updated in #237 as FILE_CONSTRAINTS is more specific, it replaced the MESSAGE_IMPORT_FOOD_FAILURE instead. But that is for another pr x.x

Reply:
Ok I put it back